app/workers/new_page_worker.rb

```diff
@@ -0,0 +1,29 @@
+class NewPageWorker
+  include Sidekiq::Worker
+
+  def perform(page_id)
+    page = Page.find(page_id)
+    make_screenshot(page)
+    parse_html_content(page)
+    puts "Page #{page.id} created!"
+  end
+
+  private
+
+  def make_screenshot(page)
+    image_path_to_save = Rails.root.join('tmp', "page_shot_#{page.id}.png")
```

Comment:
good, but not ideal
use Tempfile http://thelazylog.com/working-with-temp-files-in-ruby/
with close and unlink
Tempfile guarantee that file will be deleted, and it's stored in /tmp folder

Reply:
Gem Gastly automatically create a file in Rails root folder. How can I use Tempfile for it?

Reply:
Where in the documentation it says - about root folder?

I see following - `Gastly.capture('http://google.com', 'output.png')`

[1] pry(main)> f = Tempfile.new
=> #<File:/tmp/20180105-26015-1r2we39>
[2] pry(main)> f.path
=> "/tmp/20180105-26015-1r2we39"

SO

I can put ANY path

`Gastly.capture('http://google.com',  f.path)`